OTRExporter/extract_assets.py

```diff
@@ -18,23 +47,84 @@ def BuildOTR(xmlPath):
         print("Aborting...", file=os.sys.stderr)
         print("\n")
 
+def checkChecksum(rom):
+    r = open(rom, "rb")
+    r.seek(16)
+    bytes = r.read(4).hex().upper()
+    r.close()
+    
+    for checksum in Checksums:
+        if (checksum.value == bytes):
+            
+            for compat in CompatibleChecksums:
+                if (checksum.name == compat.name):
+                    print("Compatible rom found!")   
+                    return checksum
+            print("Valid oot rom found. However, not compatible with SoH.")
+            print("Compatible roms:")
+            for compat in CompatibleChecksums:
+                print(compat.name+" | 0x"+compat.value)
+            input("Press Enter to quit the program")
+            sys.exit(1)
+            
+    print("Wrong rom! No valid checksum found")
+    input("Press Enter to quit the program")
+    sys.exit(1)
+
 def main():
     parser = argparse.ArgumentParser(description="baserom asset extractor")
     parser.add_argument("-v", "--version", help="Sets game version.")
     args = parser.parse_args()
     
-    # TODO: Read from makerom file to automatically determine game version
-    xmlVer = "GC_NMQ_D"
-
-    if (args.version == "gc_pal_nmpq"):
-        xmlVer = "GC_NMQ_PAL_F"
-    elif (args.version == "dbg_mq"):
-        xmlVer = "GC_MQ_D"
+    romToUse = "";
+    
+    for file in glob.glob("*.z64"):
+        roms.append(file)
+        
+    if not (roms):
+        print("Error: No roms located, place one in the OTRExporter directory", file=os.sys.stderr)
+        input("Press Enter to quit the program")
+        sys.exit(1)
+        
+    if (len(roms) > 1):
```

Comment:
Could you add in a command line argument to always use `rom[0]`? I imagine there are going to be cases where the user wants to avoid providing additional input.

Reply:
If only 1 rom is in the directory no input is required. Line 112. But you mean if there are like 5 roms and I put 3 as a commandline input it chooses that one without requiring input?

Reply:
>  But you mean if there are like 5 roms and I put 3 as a commandline input it chooses that one without requiring input?

Yes, more or less.